Samples/Basic/package.json

```diff
@@ -12,7 +12,10 @@
 
   },
   "dependencies": {
-    "@dolittle/sdk": "1.2.0"
+    "@dolittle/sdk": "1.2.0",
+    "@dolittle/sdk.artifacts": "1.2.0",
+    "@dolittle/sdk.events": "1.2.0",
+    "@dolittle/sdk.events.handling": "1.2.0"
```

Comment:
The @dolittle/sdk package is an aggregated package containing all of the below. So we could go by just importing that. But that is perhaps considered bad practice and we should actually include all our dependencies. Was that the reasoning?

Reply:
The reason is that the linter was complaining, so I had to do that. As for the reason why the linter complains, you should know more than me, but I expect it is because we use those packages directly, and if for some reason `@dolittle/sdk` stopped including the others, it would break. I don't really care that much, but if we don't want to do it this way, we should remove the linter rule.

Reply:
Yeah I noticed that as well, we could just override that rule for some files / projects or disable it complete (though it is nice to have for external packages) It is more clear to have all dependencies in the projects, especially for libraries like the sdk

Reply:
But maybe, since we are specific about versions, we don't need this rule? Nothing should break unless we update a version. In which case the build should fail...

Reply:
I'll leave this to a more general discussion for later, as it applies to all of our TS projects, not just this one.